src/noggit/scripting/scripting_tool.hpp

```diff
@@ -0,0 +1,124 @@
+// This file is part of Noggit3, licensed under GNU General Public License (version 3).
+#pragma once
+
+#include <noggit/tool_enums.hpp>
+#include <math/trig.hpp>
+#include <math/vector_3d.hpp>
+
+#include <QtWidgets/QWidget>
+#include <QtWidgets/QComboBox>
+#include <QtWidgets/QLabel>
+#include <QtWidgets/QPlainTextEdit>
+#include <QtWidgets/QDoubleSpinBox>
+#include <QtWidgets/QLineEdit>
+#include <QtWidgets/QGroupBox>
+#include <QtWidgets/QFormLayout>
+#include <QtWidgets/QPushButton.h>
+#include <QtWidgets/QGridLayout.h>
```

Comment:
Qt's convention is to omit the `.h` suffix. my distribution of Qt seems to enforce this (Arch Linux, Qt 5.15.2) and thus fails to build